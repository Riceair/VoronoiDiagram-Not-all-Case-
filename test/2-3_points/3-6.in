#三點測試 y=2x 三點共線
3
10 20
20 40
200 400