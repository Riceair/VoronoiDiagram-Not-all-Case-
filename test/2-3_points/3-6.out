P 10 20
P 20 40
P 200 400
E -226690 113620 226910 -113180
E -12585 6330 12615 -6270
